pkg/util/dns/dns.go

```diff
@@ -18,7 +18,10 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/dns"
 )
 
-const resourceID = "resourceId"
+const (
+	resourceID          = "resourceId"
+	managedDomainSuffix = ".aroapp.io"
+)
```

Comment:
This doesn't take into account the govcloud domains.  Can we update this to switch on the ARO cluster CR based on which cloud we are in?

For reference, in gov cloud the `managedDomainSuffix` would be: `aroapp.azure.us`

Reply:
Thanks @bennerv! Good catch. Doing it now.